geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/NullRedisSortedSet.java

```diff
@@ -44,10 +43,16 @@ public boolean isNull() {
   long zadd(Region<RedisKey, RedisData> region, RedisKey key, List<byte[]> membersToAdd,
       ZAddOptions options) {
     if (options.isXX()) {
-      return 0;
+      return 0L;
+    }
+
+    for (int i = 0; i < membersToAdd.size(); i += 2) {
+      byte[] incr = processIncrement(Coder.bytesToString(membersToAdd.get(i)));
+      membersToAdd.set(i, incr);
```

Comment:
We already loop through the members and scores in the `RedisSortedSet` constructor and call `processIncrement` on the score.  There is no need to do it twice.